pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -88,6 +89,21 @@ message Metadata {
 
 // Record corresponds to a protected timestamp.
 message Record {
+  message SchemaObjectsTarget {
+    // IDs are the descriptor IDs of the schema objects being protected by this
+    // Record. This field will only contain database and table IDs.
+    repeated uint32 ids = 1 [(gogoproto.customname) = "IDs",
+      (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+  }
+
+  message TenantsTarget {
+    // IDs are the tenant IDs being protected by this Record.
+    repeated roachpb.TenantID ids = 1 [(gogoproto.customname) = "IDs"];
+  }
+
+  message ClusterTarget {
+    // Indicates that the entire cluster is being protected by this Record.
```

Comment:
[nit] Maybe it's clearer to others, but "cluster" feels vague, adding more commentary would help (either referring to FULL CLUSTER backups explicitly or saying it protects all SQL state; for secondary tenants it implies just their own stuff, for hosts it includes both its own and all secondary tenants').

Reply:
good point, done.